plugin/src/main/scala/org/virtuslab/akkasaferserializer/SaferSerializerPlugin.scala

```diff
@@ -0,0 +1,22 @@
+package org.virtuslab.akkasaferserializer
+
+import scala.tools.nsc.Global
+import scala.tools.nsc.plugins.{Plugin, PluginComponent}
+
+class SaferSerializerPlugin(override val global: Global) extends Plugin {
+  override val name: String = "safer-serializer-plugin"
+  override val description: String = "checks for Behavior[A] and ActorRef[A]"
```

Comment:
I'd make it the description more generic, since there'll be surely many more classes to consider: maybe sth like `checks that the specified Akka serialization is applied to all messages, event and persistent state classes`